src/components/Scorecard.vue

```diff
@@ -35,8 +35,10 @@
           class="relative mx-auto w-full flex justify-center"
           :radius="circularProgressRadius"
           :stroke="circularProgressStroke"
-          :result="progressBarResult"
+          :result="result"
+          :progressBarPercent="localProgressBarPercentage"
```

Comment:
```suggestion
          :progressBarPercentage="localProgressBarPercentage"
```

Keeping names on both sides consistent